src/Support/Composer.php

```diff
@@ -44,18 +44,27 @@ final class Composer
      */
     private $loggingPath;
 
+    /**
+     * The composer location  path.
+     *
+     * @var string
+     */
+    private $composerPath;
+
     /**
      * Create a new Composer manager instance.
      *
      * @param \Illuminate\Filesystem\Filesystem $files
      * @param string $workingPath
      * @param string $loggingPath
+     * @param string $composerPath
      */
-    public function __construct(Filesystem $files, string $workingPath, string $loggingPath)
+    public function __construct(Filesystem $files, string $workingPath, string $loggingPath, string $composerPath = '/usr/bin/composer')
```

Comment:
Remove $composerPath default value here.
`string $composerPath` will be enough

Reply:
ok, Removed